cli/dcoscli/package/main.py

```diff
@@ -502,8 +505,21 @@ def _uninstall(package_name, remove_all, app_id, cli, app):
     """
 
     package_manager = _get_package_manager()
-    err = package.uninstall(
-        package_manager, package_name, remove_all, app_id, cli, app)
+
+    if(force == True):
+        toml_config = config.get_config()
+        dcos_url = config.get_config_val("core.dcos_url", toml_config)
+        master_url = "{0}:5050/master".format(dcos_url)
+        exhibitor_url = "{0}:8181".format(dcos_url)
+
+        janitor = janitorpackage.Janitor(package_manager, emitter)
+
+        janitor.destroy_volumes(master_url, "{0}-role".format(package_name), "{0}-principal".format(package_name))
+        janitor.unreserve_resources(master_url, "{0}-role".format(package_name), "{0}-principal".format(package_name))
```

Comment:
The principal and role may be customized by the user (beyond just the above variance based on the service name), so it should be possible for the user to optionally specify any custom role/principal when running the command.